ts/components/ChooserListItem.tsx

```diff
@@ -1,11 +1,18 @@
-import { Text, View } from "native-base";
+import { Button, Text, View } from "native-base";
 import * as React from "react";
-import { StyleSheet } from "react-native";
+import { StyleSheet, TouchableOpacity } from "react-native";
+import variables from "../theme/variables";
 import customVariables from "../theme/variables";
+import IconFont from "./ui/IconFont";
 
 type Props = Readonly<{
-  title: string;
-  iconComponent?: React.ReactElement;
+  itemTitle: string;
+  itemId: string;
+  itemIconComponent?:
```

Comment:
A cleaner solution could be [Either](https://gcanti.github.io/fp-ts/modules/Either.ts.html)?

Reply:
Done in commit 1a7b88d